packages/core/src/components/dropdown/dropdown.stories.tsx

```diff
@@ -185,6 +185,7 @@ const sizeLookUp = {
   Large: 'lg',
   Medium: 'md',
   Small: 'sm',
+  'Extra small': 'xs',
```

Comment:
Minor thing but we tend to do differently here, sometimes all is in quotes, sometimes without. But maybe not mix?

Reply:
then i suggest `Xsmall` as name there